packages/application-shell/src/components/locale-switcher/messages.ts

```diff
@@ -0,0 +1,9 @@
+import { defineMessages } from 'react-intl';
+
+export default defineMessages({
+  localesLabel: {
+    id: 'LocaleSwitcher.localesLabel',
+    description: 'The label for project dropdown switcher',
+    defaultMessage: 'Locales menu',
```

Comment:
I used the `menu` word instead of `switcher` or `selector` as that's the word we use for the user menu dropdown.
Let me know if you think we should change it for the switchers.

Reply:
Remember to run this command to extract the new messages:

https://github.com/commercetools/merchant-center-application-kit/blob/6cf96ee64d59177675a1d0bdea9a1ef953596198/package.json#L9

Reply:
Thanks for the reminder.

Updated here: 212e4d4e65f410075bf84a2090ef428372995829